src/DriverFactory.php

```diff
@@ -42,8 +37,6 @@ protected function createWithConfig(ContainerInterface $container, $configKey)
         }
 
         if ($config['class'] === AnnotationDriver::class || is_subclass_of($config['class'], AnnotationDriver::class)) {
-            $this->registerAnnotationLoader();
```

Comment:
Not hundered percent sure on this. `AnnotationRegistry::registerLoader()` was deprecated ~3 years ago.

Reply:
I vaguely remember that there's not really an alternative for now, and if we're still supporting ancient Doctrine versions, this probably needs to stay I think. Is there a way to detect if we need to register the annotation loader I wonder? 🤔

Reply:
In fact, further to this, since we moved to PHP 7.3 I expect we can bump the minimum Doctrine we support too, but I'd need to investigate what version that would be exactly (or you can, I don't mind)

Reply:
> I vaguely remember that there's not really an alternative for now

Now I'm more awake, I remember that the alternative is "the autoloader should load this" :sweat_smile:

Reply:
Do you mean its fine to not registering any loader in this library's scope and removing the call as I proposed?

Reply:
I mean I'm unsure; older versions of Doctrine still need this don't they?